apps/backoffice/src/components/services/service-preview.tsx

```diff
@@ -44,10 +44,12 @@ export const ServicePreview = ({
 
   return (
     <Dialog open={isDialogOpen} onClose={handleClose} disableScrollLock>
-      <DialogTitle>{t("service.preview.title")}</DialogTitle>
+      <DialogTitle id="title">{t("service.preview.title")}</DialogTitle>
```

Comment:
it is safe put as id a generic "title"? I mean can this id be erronuosly used in other elements cause is too generic and cause some issue?

Reply:
[9914d3f](https://github.com/pagopa/io-services-cms/pull/503/commits/9914d3f427282d776af4b7a8f9e6cbc08cfcec21)